### DISCUSSION TITLE
Need to update the version detail in a yaml file

### DISCUSSION THREAD

**Discussion by kaushique** [renovatebot/renovate#32527] [Request Help]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

I'm trying to update the Fiori version in the yaml file of type application . Since Fiori is not listed in npm repos or registry , I'm not able to provide the valid datasources name or depName . I'm sharing the ui5.yaml in tst format and renovate.json that I'm using . Please let me know where I'm going wrong with proper guidance. 
[renovate.json](https://github.com/user-attachments/files/17744029/renovate.json)
[ui5.txt](https://github.com/user-attachments/files/17744069/ui5.txt)

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
You've provided insufficient information or effort for us to help you. Please don't just dump your work on open source projects and expect free help without putting in maximum effort yourself first and then describing where you're blocked. This is not a forum for you to say "I tried, it didn't work, please finish it for me"
